index.html

```diff
@@ -81,92 +94,46 @@ <h1 class="section-title hero-banner__title">
             </div>
-          </article>
-          <article class="work-card">
-            <div class="work-card__head"></div>
-            <div class="work-card__body">
-              <h3 class="card-title">Multi-Post Stories Gain+Glory</h3>
-              <ul class="card-btns">
-                <li class="card-btn">Ruby on Rails</li>
-                <li class="card-btn">CSS</li>
-                <li class="card-btn">Javascript</li>
-                <li class="card-btn">HTML</li>
-              </ul>
-              <div class="cta-btn"><a href="#">See project</a></d>
-            </div>
-          </article>
-          <article class="work-card">
-            <div class="work-card__head"></div>
-            <div class="work-card__body">
-              <h3 class="card-title">Multi-Post Stories Gain+Glory</h3>
-              <ul class="card-btns">
-                <li class="card-btn">Ruby on Rails</li>
-                <li class="card-btn">CSS</li>
-                <li class="card-btn">Javascript</li>
-                <li class="card-btn">HTML</li>
-              </ul>
-              <div class="cta-btn"><a href="#">See project</a></d>
-            </div>
-          </article>
-          <article class="work-card">
-            <div class="work-card__head"></div>
-            <div class="work-card__body">
-              <h3 class="card-title">Multi-Post Stories Gain+Glory</h3>
+            <div class="popup-body">
+              <h3 class="project-popup__title">Keeping track of hundreds of
+                components</h3>
               <ul class="card-btns">
                 <li class="card-btn">Ruby on Rails</li>
                 <li class="card-btn">CSS</li>
                 <li class="card-btn">Javascript</li>
                 <li class="card-btn">HTML</li>
               </ul>
-              <div class="cta-btn"><a href="#">See project</a></d>
+              <div class="project-description">
+                <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.</p>
+                <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.</p>
+              </div>
+              <div class="project-popup-btns">
+                <div class="project-popup-btn">
+                  <a href="#">See live</a>
+                  <img src="images/icon-export.png" alt="export icon">
+                </div>
+                <div class="project-popup-btn">
+                  <a href="#">See source</a>
+                  <img src="images/icon-github.png" alt="github icon">
+                </div>
+              </div>
             </div>
```

Comment:
- [ ] Nice work implementing the design, everything looks awesome :clap: :sunglasses: However it would be nice to remove this html popup code from your code base since you're not using it. Doing that will make your code clean and DRY